rts/Game/UI/SelectionKeyHandler.cpp

```diff
@@ -24,6 +22,7 @@
 #include "System/UnorderedSet.hpp"
 
 CSelectionKeyHandler selectionKeys;
+std::unordered_set<std::string> idMatchesSet;
```

Comment:
We don't have to allocate every invocation right? Although we could make it static I guess inside

Reply:
I think the perf differences are irrelevant and I'm suggesting a local because globals/statics make maintenance harder (which in this case isn't going to be terribly relevant either, but just on a principle)